src/sentry/models/group.py

```diff
@@ -377,6 +377,39 @@ def get_latest_event(self):
                 self._latest_event = None
         return self._latest_event
 
+    def get_latest_event_for_environments(self, environments=[]):
+        from sentry.utils import snuba
+        from sentry.models import SnubaEvent
+
+        use_snuba = options.get('snuba.events-queries.enabled')
```

Comment:
I assume it's fine to use this particular option here to detect whether snuba is enabled

Reply:
yes, although this might just be turned off in prod?

Reply:
Oh. What would be the best way to detect whether Snuba is available then?

Reply:
Yeah I think this is the right flag... Just not sure if anyone has switched it on yet